tests/form_autofill/test_edit_credit_card.py

```diff
@@ -0,0 +1,129 @@
+import json
+import logging
+
+import pytest
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+from modules.browser_object import Navigation
+from modules.browser_object_autofill_popup import AutofillPopup
+from modules.page_object import AboutPrefs
+from modules.page_object_autofill_credit_card import CreditCardFill
+from modules.util import BrowserActions, Utilities
+
+tabs = [i for i in range(4)]
+
+
```

Comment:
Would love to have `@pytest.mark.unstable` here, just because we have a known (though very low) amount of flake.